SplunkADXForwarder/app.py

```diff
@@ -0,0 +1,247 @@
+import socket
+import logging
+import time
+import os
+import yaml
+import pandas as pd
+import csv
+import threading
+import tempfile
+import shutil
+from azure.kusto.data import KustoConnectionStringBuilder
+from azure.kusto.data.data_format import DataFormat
+from azure.kusto.ingest import (
+    IngestionProperties,
+    IngestionStatus,
+    QueuedIngestClient
+)
+
+# Constants
+MAX_BATCH_SIZE = int(os.environ.get('MAX_BATCH_SIZE', '1000'))
```

Comment:
when receiving incoming data from a socket, the MAX_BATCH_SIZE typically refers to the maximum amount of data (in bytes) that our application should receive in a single read or receive operation. In our case, MAX_BATCH_SIZE can determine how much data the application waits to receive before processing it. If the received data size is less than MAX_BATCH_SIZE, the application may block (wait) until it receives enough data or until the connection is closed.  Since this was internally used for data read, hence didnt add to ReadMe.